frontend/src/components/navbar/NavBar.jsx

```diff
@@ -10,6 +11,9 @@ function NavBar(props) {
     <header>
       <img className="icon" src={logo} alt="The logo" />
       <Search setVille={setVille} />
+      <button className="button_filter" type="button">
```

Comment:
Keep your code convention for css classes and rename it in buttonFilter, don't forget to rename it in css.